### PR TITLE
ticdc: add HTTP API for adjusting log level

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

ticdc: add HTTP API for adjusting log level

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4560
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Review by amyangfei — COMMENTED** [review]
LGTM

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -596,6 +596,16 @@ For nodes other than owner nodes, executing the above command will return the fo
...
 
+### Dynamically change the log level of TiCDC server
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X POST -d '"debug"' http://127.0.0.1:8301/admin/log
+```
+
+In the command above, the `POST` parameter indicates the new log level. The [zap-provided](https://godoc.org/go.uber.org/zap#UnmarshalText) log level options are supported: "debug", "info", "warn", "error", "dpanic", "panic", and "fatal". This interface parameter is JSON-encoded and you need to pay attention to the use of quotation marks. For example: `"debug"`.
```
Comment (yikeke) [inline ticdc/manage-ticdc.md:607]: The original [Chinese version](https://github.com/pingcap/docs-cn/pull/4560/files#diff-c838c41b0017f0b0cb1d5bb0a42dcfccR594) is ``` `'"debug"'` ```, which one is wrong? @TomShawn @amyangfei

Comment (amyangfei) [reply ticdc/manage-ticdc.md:607]: should be `` `'"debug"'` ``

Comment (TomShawn) [inline ticdc/manage-ticdc.md:607]:
```suggestion
In the command above, the `POST` parameter indicates the new log level. The [zap-provided](https://godoc.org/go.uber.org/zap#UnmarshalText) log level options are supported: "debug", "info", "warn", "error", "dpanic", "panic", and "fatal". This interface parameter is JSON-encoded and you need to pay attention to the use of quotation marks. For example: `'"debug"'`.
```

**Review by yikeke — APPROVED** [review]
LGTM

File: ticdc/manage-ticdc.md (hunk 1 above)
Comment (TomShawn) [reply ticdc/manage-ticdc.md]: Fixed in https://github.com/pingcap/docs/pull/4005/commits/68c5561ff4eef1c30bd27c0c64093ff3bee36103.

**Pull request merged**